### PR TITLE
Waggle - separate unit tests that use tor - they take a long time and…

### PR DESCRIPTION
**Pull request by EmiM**
… are less stable

### CODE REVIEW

File: packages/waggle/src/libp2p/connectionsManager.test.ts [hunk 1]
```diff
@@ -16,25 +15,6 @@ beforeEach(() => {
 })
 
 describe('Connections manager', () => {
```
Comment (siepra) [inline packages/waggle/src/libp2p/connectionsManager.test.ts:17]: Maybe add some info that those tests runs without tor? (not using tor)

File: packages/waggle/src/registration/testUtils.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import fetch, { Response } from 'node-fetch'
+import PeerId from 'peer-id'
+import { CertificateRegistration } from '.'
+import { createLibp2p } from '../common/testUtils'
+import { DataFromPems } from '../common/types'
+import { DummyIOServer } from '../common/utils'
+import { Storage } from '../storage'
+import { Tor } from '../torManager'
+
+export async function registerUserTest(csr: string, httpTunnelPort: number, localhost: boolean = true, registrarPort: number = 7789): Promise<Response> {
```
Comment (siepra) [inline packages/waggle/src/registration/testUtils.ts:11]: registerUserTest sounds like it's a test scenario

**Review by siepra — APPROVED** [review]
I think it's ok. A little bit more complicated than it was before but we all understand why it has to be done that way

File: packages/waggle/src/libp2p/connectionsManager.test.ts (hunk 1 above)
Comment (EmiM) [reply packages/waggle/src/libp2p/connectionsManager.test.ts:17]: I think that's self explanatory if we have `.tor.test.ts` files with tor tests. We would have to add "(not using tor)" to 95% of our test cases.
I added (using tor) to tor-test names instead.

**Pull request merged**